README.md

```diff
@@ -1,6 +1,6 @@
 # Address Book (Level 4)
 
-[![Build Status](https://travis-ci.org/se-edu/addressbook-level4.svg?branch=master)](https://travis-ci.org/se-edu/addressbook-level4)
+[![Build Status](https://travis-ci.org/CS2103JAN2017-T15-B1/main.svg?branch=master)](https://travis-ci.org/CS2103JAN2017-T15-B1/main)
```

Comment:
- similar to this, you need to update the other badges (coverall, codacy etc.,) to reflect to your project.
- in the above, the title should reflect your product not address book anymore!